src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -45,80 +43,66 @@ protected String getPackageLocation() {
 
     @Test
     public void testMissing() throws Exception {
-        final Configuration checkConfig = createModuleConfig(JavadocPackageCheck.class);
         final String[] expected = {
-            "1: " + getCheckMessage(MSG_PACKAGE_INFO),
+            "9: " + getCheckMessage(MSG_PACKAGE_INFO),
         };
-        verify(
-            checkConfig,
+        verifyWithInlineConfigParser(
             getPath("InputJavadocPackageBadCls.java"),
             expected);
     }
 
     @Test
     public void testMissingWithAllowLegacy() throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(JavadocPackageCheck.class);
-        checkConfig.addProperty("allowLegacy", "true");
         final String[] expected = {
-            "1: " + getCheckMessage(MSG_PACKAGE_INFO),
```

Comment:
@shubh220922 since we cannot alter the check, this number will not change. Please try the following:
1. Revert changes to expected in all tests
2. Make violation comments like `// violation 9 lines above 'Missing package-info.java file'`

Reply:
This check is basically placing a violation on the first line of the given file, so you can just put the violation comment wherever is appropriate and make the comment point to line 1.

Reply:
Yes, `// violation x lines above `
Right after config comment